frameworks/svelte/__tests__/responsive.test.ts

```diff
@@ -26,9 +26,9 @@ describe('responsive', () => {
       advancedImgProps: {plugins: [responsive()]}
     });
 
-    let el = dispatchResize(container, 100);
+    const el = dispatchResize(container, 100);
     await waitForExpect(() => {
-      expect(el.clientWidth).toBe(100);
+      expect(el && el.clientWidth).toBe(100);
```

Comment:
Why do we need to check in the `expect` itself that `el` is truthy?

Reply:
Otherwise you get a warning that el might be null

Reply:
It sounds like the type of `dispatchResize` needs to be improved.

This might help

```
import {ReactWrapper} from "enzyme";

export function dispatchResize(component: ReactWrapper, resizeValue: number):  { /*... */ }

```

You can also explicitly add the export type 
```
function foo(): Element {
 return  // Anything returned HAS to be an Element
}
```